managed/testdata/pg/Makefile

```diff
@@ -26,9 +26,11 @@ server-ssl:                                          ## Generates server-ssl cer
 		-out /tmp/server.csr
 	openssl x509 -req -days 3653 -sha256 \
 		-extfile ./certs/server.cnf -extensions req_ext \
-		-CA ./certs/root.crt -CAkey /tmp/root.key -CAcreateserial \
+		-CA ./certs/root.crt -CAkey ./certs/root.key -CAcreateserial \
 		-in /tmp/server.csr \
 		-out ./certs/server.crt
+	chmod 600 certs/server.crt
+	chmod 600 certs/server.key
 
```

Comment:
if this is done in order to save the file mod in the `git`, then you will need to do [this](https://stackoverflow.com/questions/9027584/how-to-change-the-file-mode-on-github) additionally, so it maybe better to change file mod on copying to the container in one line, as it done in [Dockerfile](https://github.com/percona/pmm/blob/main/managed/testdata/pg/Dockerfile#L6-L7) or [here](https://github.com/percona/pmm/blob/main/docker-compose.yml#L93-L101)